src/server/page_server.cpp

```diff
@@ -163,6 +167,19 @@ page_server::connection_handler::receive_data_page_fetch (tran_server_conn_t::se
   push_async_response (&pgbuf_respond_data_fetch_page_request, std::move (a_sp));
 }
 
+void
+page_server::connection_handler::receive_oldest_active_mvccid_request (tran_server_conn_t::sequenced_payload &a_sp)
```

Comment:
The function also does the handling.
I know that others - like `receive_stop_log_prior_dispatch` - are also mis-named but we'll fix them sometime.
```suggestion
page_server::connection_handler::handle_oldest_active_mvccid_request (tran_server_conn_t::sequenced_payload &a_sp)
```